pages/index.html

```diff
@@ -1,130 +1,171 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" xmlns="http://www.w3.org/1999/html">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Concept Website</title>
     <link href="styles.css" rel="stylesheet">
 </head>
 
 <body>
-
-
-
-    <header>
+<header class="outer-container">
+    <div class="inner-container">
         <nav>
-            <ul></ul>
-            <ul></ul>
-            <ul></ul>
+            <ul>
+                <li><a href="#image-area">Case Studies</a></li>
+                <li><a href="#service-area">Services</a></li>
+                <li><a href="#form-area">Contact Us</a></li>
+            </ul>
         </nav>
 
-        <div>
-            <h1></h1>
-            <h3></h3>
-            <button type="submit"></button>
+        <div class="header-text">
+            <h1>Your website is an investment in the success of your business.</h1>
+            <h3>We are here to help you succeed.</h3>
+            <button type="button" name="first-pinky-button" class="button">Let's work together</button>
         </div>
 
-        <img src="" alt="">
-    </header>
-
-    <main>
-        <section>
-            <div>
-                <h2></h2>
-                <p></p>
+        <img src="/assets/squiggle_line_stripe.svg" alt="squiggle-line" class="drawing"/>
```

Comment:
Voor alle plaatjes geldt dat ze een folder hoger staan
```suggestion
        <img src="../assets/squiggle_line_stripe.svg" alt="squiggle-line" class="drawing"/>
```